app/setup.py

```diff
@@ -160,6 +164,18 @@ def override_url_for():  # pylint: disable=unused-variable
     return application
 
 
+def setup_secure_headers(application):
+    Talisman(
+        application,
+        content_security_policy=CSP_POLICY,
+        content_security_policy_nonce_in=['script-src'],
+        session_cookie_secure=not application.config['EQ_SECURE_SESSION_COOKIE'],
+        force_https=False,  # this is handled at the firewall
+        strict_transport_security=True,
+        strict_transport_security_max_age=31536000,
```

Comment:
Do you not want to includeSubDomains for this?

Reply:
Yep that's included as the default HSTS value.